syntax = "proto3";

package face_aggregation;

import "common.proto";

service AgenderAnalysisService {
  rpc ReceiveImage(ImageToAgenderServiceRequest) returns (DoneFlagToImageInputServiceResponse);
}